packages/patternfly-4/react-core/src/experimental/components/SimpleList/SimpleList.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/SimpleList/simple-list';
+import { SimpleListGroup } from './SimpleListGroup';
+
+export interface SimpleListProps {
+  /** Content rendered inside the SimpleList */
+  children?: React.ReactNode;
+  /** Additional classes added to the SimpleList <ul> */
+  className?: string;
+  /** Callback when an item is selected */
+  onSelect?: (ref: React.RefObject<HTMLButtonElement> | React.RefObject<HTMLAnchorElement>) => void;
```

Comment:
Instead of providing the button associated with clicking on each menu option, I'd prefer to provide my own ID. I can get the label from the button, but it's difficult to work with considering the label may be localized.

![Screen Shot 2020-02-05 at 10 59 26 AM](https://user-images.githubusercontent.com/17481322/73858757-e8be2300-4806-11ea-922b-a7b85065c39f.png)
...
![Screen Shot 2020-02-05 at 10 59 46 AM](https://user-images.githubusercontent.com/17481322/73858767-eb207d00-4806-11ea-9630-e29aff47a8b2.png)

@dlabaj I believe we may be using `SelectOption` to help with this?